app/developers/routes.py

```diff
@@ -511,7 +511,9 @@ def edit_question(
 @platform_admin_role_required
 def collection_tasklist(collection_id: UUID) -> ResponseReturnValue:
     collection_helper = CollectionHelper.load(collection_id)
-    return render_template("developers/collection_tasklist.html", collection_helper=collection_helper)
+    return render_template(
+        "developers/collection_tasklist.html", collection_helper=collection_helper, statuses=CollectionStatusEnum
```

Comment:
possibly keep an eye on whether we inject CollectionStatusEnum - and some other enums - automatically in the future. Fine for now.